azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -299,6 +307,39 @@ public void uploadLogFile(final int execId, final String name, final int attempt
     this.executionLogsDao.uploadLogFile(execId, name, attempt, files);
   }
 
+  @Override
+  public void appendLogs(final int execId, final String name, final String logs) throws
+      ExecutorManagerException {
+    if (StringUtils.isBlank(logs)) {
+      return;
+    }
+    File tempFile = null;
+    try {
+      tempFile = File.createTempFile("appendLogs-" + execId + "-" + name, ".tmp");
+      Files.write(tempFile.toPath(), logs.getBytes(StandardCharsets.UTF_8));
+      this.executionLogsDao.appendLogs(execId, "", 0, tempFile);
+      Thread.sleep(1000);
```

Comment:
Why is this sleep needed ?

Reply:
Is this sleep added for debugging? I believe so.

Reply:
Yes, thank you.